plugins/spark/spark-env.install

```diff
@@ -0,0 +1,25 @@
+
+mkdir -p ${CONFIG_DIR}/rpmnew
+mkdir -p ${CONFIG_DIR}/externals/spark-hadoop
+
+printf "Installing %-44s ..." "spark-env.sh"
+
+if [ ! -e ${CONFIG_DIR}/spark-env.sh ]; then
+    # Always install new files without comment
+    cp -f ${INSTALL_DIR}/externals/spark-hadoop/conf/spark-env.sh ${CONFIG_DIR}/externals/spark-hadoop/spark-env.sh
```

Comment:
Has spark-env.sh not been renamed to spark-hpcc-env.sh? Should this line (and various others) therefore change? Should this file spark-env.install also be renamed?

Reply:
@richardkchapman 

There are two files now that we use to source in variables for the spark environment.  The one we generate using xslt according to what we have in the environment.xml file is now going to be called 'spark-hpcc-env.sh' and located under /var/lib/HPCCSystems/<componentName>.  This is the file that's being renamed.  We're changing it because the user can't really modify that file unless they modify the xslt file that generates it.  So in documentation when we say 'you can modify the spark-env.sh file', there should be less confusion because we don't have two files named the same thing.

The one that the user can change will be called 'spark-env.sh' and located under /etc/HPCCSystems/externals/spark-hadoop/.